integrationtest/spring/src/test/resources/persistence.xml

```diff
@@ -17,7 +17,7 @@
     </persistence-unit>
 
     <persistence-unit name="jbossjta" transaction-type="JTA">
-        <provider>org.hibernate.ejb.HibernatePersistence</provider>
+        <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
```

Comment:
thanks! I had actually changed that but then sent the wrong branch .. good catch! Merging the right branch..